### ISSUE TITLE
Accessibility Testing Components

### ISSUE THREAD

**Issue by Blackbaud-BrandonJones** [blackbaud/stache2#478]
Our tests should be updated for each component to pass the accessibility tests from the `skyux-lib-testing` library.  https://github.com/blackbaud/skyux-lib-testing#accessibility-testing

```
  it('should be accessible', async(() => {
    expect(fixture.debugElement.nativeElement).toBeAccessible();
  }));
```
We should also update the components and layouts to ensure all our content is manually tested for accessibility standards as well.
